src/assets/styles/theme.scss

```diff
@@ -70,6 +70,7 @@ $app-dk-blue: #38598a; // same as the Vuetify theme appDkBlue
 $app-red: #d3272c;
 $app-green: #1a9031;
 $app-alert: #F8661A;
+$app-tooltip-color: #3b6cff;
```

Comment:
Is this variable used anywhere?

Also, "color" in the name is redundant :) How about just app-tooltip?

Reply:
No actually, it's not. I just wanted to be consistent with the other declarations in `main.ts`. Also, agreed.

I think I'll just remove the entire declaration. In case we need it down the line, we can re-add it.